packages/xsnap-lockdown/scripts/build-bundle.js

```diff
@@ -28,7 +28,20 @@ const make = async name => {
   return { length: bundleString.length, sha256, where: spec };
 };
 
+/**
+ * @param {string} name
+ */
+const makeObjectInspect = async name => {
+  const spec = bundlePaths[name];
+  const entryPath = entryPaths[name];
+  await fsp.mkdir(path.dirname(spec), { recursive: true });
+  const bundle = await bundleSource(entryPath, { format: 'getExport' });
+  const bundleString = JSON.stringify(bundle);
+  await fsp.writeFile(spec, `export default ${bundleString};\n`);
+};
```

Comment:
Suggestion: rather than introducing a new function, generalize `make`.
```js
const make = async (name, format) => {
  const spec = bundlePaths[name];
  const entryPath = entryPaths[name];
  const hashPath = hashPaths[name];
  await fsp.mkdir(path.dirname(spec), { recursive: true });
  const bundle = await bundleSource(entryPath, { format });
  const bundleString = JSON.stringify(bundle);
  const contents =
    format === 'getExport' ? `export default ${bundleString};\n` : bundleString;
  await fsp.writeFile(spec, contents);
  let sha256;
  if (hashPath) {
    sha256 = computeSha256(contents);
    await fsp.writeFile(hashPath, `${sha256}\n`);
  }
  return { length: contents.length, sha256, where: spec };
};
```

Reply:
I am passing through a phase where I favor the clarity of linear copy pasta in some cases over branchy desiccated logic. Since you don’t insist, I’m going to leave it here until it becomes a nuisance.